app/registration/src/views/registration/MemberRecordsView.vue

```diff
@@ -13,19 +13,24 @@
             <h2>Privacy</h2>
 
             <Checkbox v-if="privacyUrl" v-model="allowData" class="long-text">
-                Ik geef toestemming aan {{ organization.name }} om de gevoelige gegevens van {{ memberDetails.firstName }}, dewelke ik hieronder kan vermelden, te verzamelen en te verwerken. Hoe we met deze gegevens omgaan staat vermeld in <a class="inline-link" :href="privacyUrl" target="_blank">het privacybeleid</a>.
+                Ik geef toestemming aan {{ organization.name }} om de gevoelige gegevens van {{ memberDetails.firstName }}, dewelke ik hieronder kan vermelden, te verzamelen en te verwerken (o.a. voor medische steekkaart). Hoe we met deze gegevens omgaan staat vermeld in <a class="inline-link" :href="privacyUrl" target="_blank">het privacybeleid</a>.
             </Checkbox>
             <Checkbox v-else v-model="allowData" class="long-text">
-                Ik geef toestemming aan {{ organization.name }} om de gevoelige gegevens van {{ memberDetails.firstName }}, dewelke ik hieronder kan vermelden, te verzamelen en te verwerken.
+                Ik geef toestemming aan {{ organization.name }} om de gevoelige gegevens van {{ memberDetails.firstName }}, dewelke ik hieronder kan vermelden, te verzamelen en te verwerken (o.a. voor medische steekkaart).
             </Checkbox>
 
             <Checkbox v-model="isParent" v-if="allowData && memberDetails.age < 18" class="long-text">
                 Ik ben wettelijke voogd of ouder van {{ memberDetails.firstName }} en mag deze toestemming geven.
             </Checkbox>
-            <Checkbox v-model="allowPictures" class="long-text">
+            <Checkbox v-if="!allowGroupPictures" v-model="allowPictures" class="long-text">
```

Comment:
Dit lijntje zou ik gewoon laten. Altijd vragen of er toestemming is voor foto's => als er geen toestemming is (zie lijnen hieronder, die zijn goed) dan toon je de extra vraag om wel toestemming te geven voor groepsfoto's